include/picongpu/param/unit.param

```diff
@@ -29,7 +29,11 @@
 
 namespace picongpu
 {
-    /** Unit of time */
+    /** Unit of time
+     *
+     * @attention atomicPhysics2 assumes that UNIT_TIME == SI:DELTA_T_SI,
```

Comment:
`atomicPhysics2` is a working title do you mean `atomicPhysics`

Reply:
this comment is atomicPhysics specific therefore I will remove it from this PR, good catch.